packages/arb-bridge-eth/test/sequencerInbox.spec.ts

```diff
@@ -0,0 +1,447 @@
+      delayedTx.l1GasPrice,
+      delayedTx.senderAddr,
+      delayedTx.deliveredMessageEvent.messageDataHash,
+      delayedTx.delayedAcc
+    )
+  })
+
+  it('can force-include one after another', async () => {
+    const delayedTx = await sendDelayedTx(
+      user,
+      inbox,
+      bridge,
+      messageTester,
+      1000000,
+      21000000000,
+      0,
+      await user.getAddress(),
+      BigNumber.from(10),
+      '0x1010'
+    )
+
+    const delayedTx2 = await sendDelayedTx(
+      user,
+      inbox,
+      bridge,
+      messageTester,
+      1000000,
+      21000000000,
+      1,
+      await user.getAddress(),
+      BigNumber.from(10),
+      '0xdeadface'
+    )
+
+    const maxDelayBlocks = (await sequencerInbox.maxDelayBlocks()).toNumber()
+    await mineBlocks(user, maxDelayBlocks)
+
+    await forceIncludeMessages(
+      sequencerInbox,
+      delayedTx.inboxAccountLength,
+      delayedTx.deliveredMessageEvent.kind,
+      delayedTx.l1BlockNumber,
+      delayedTx.l1BlockTimestamp,
+      delayedTx.deliveredMessageEvent.messageIndex,
+      delayedTx.l1GasPrice,
+      delayedTx.senderAddr,
+      delayedTx.deliveredMessageEvent.messageDataHash,
+      delayedTx.delayedAcc
+    )
+    await forceIncludeMessages(
+      sequencerInbox,
+      delayedTx2.inboxAccountLength,
+      delayedTx2.deliveredMessageEvent.kind,
+      delayedTx2.l1BlockNumber,
+      delayedTx2.l1BlockTimestamp,
+      delayedTx2.deliveredMessageEvent.messageIndex,
+      delayedTx2.l1GasPrice,
+      delayedTx2.senderAddr,
+      delayedTx2.deliveredMessageEvent.messageDataHash,
+      delayedTx2.delayedAcc
+    )
+  })
+
+  it('can force-include three at once', async () => {
+    await sendDelayedTx(
+      user,
+      inbox,
+      bridge,
+      messageTester,
+      1000000,
+      21000000000,
+      0,
+      await user.getAddress(),
+      BigNumber.from(10),
+      '0x1010'
+    )
+    await sendDelayedTx(
+      user,
+      inbox,
+      bridge,
+      messageTester,
+      1000000,
+      21000000000,
+      1,
+      await user.getAddress(),
+      BigNumber.from(10),
+      '0x101010'
+    )
+    const delayedTx3 = await sendDelayedTx(
+      user,
+      inbox,
+      bridge,
+      messageTester,
+      1000000,
+      21000000000,
+      10,
+      await user.getAddress(),
+      BigNumber.from(10),
+      '0x10101010'
+    )
+
+    const maxDelayBlocks = (await sequencerInbox.maxDelayBlocks()).toNumber()
+    await mineBlocks(user, maxDelayBlocks)
+
+    await forceIncludeMessages(
+      sequencerInbox,
+      delayedTx3.inboxAccountLength,
+      delayedTx3.deliveredMessageEvent.kind,
+      delayedTx3.l1BlockNumber,
+      delayedTx3.l1BlockTimestamp,
+      delayedTx3.deliveredMessageEvent.messageIndex,
+      delayedTx3.l1GasPrice,
+      delayedTx3.senderAddr,
+      delayedTx3.deliveredMessageEvent.messageDataHash,
+      delayedTx3.delayedAcc
+    )
+  })
+
+  it('cannot include before max delay', async () => {
```

Comment:
cases covered here are solid - i'd just add some logic to check for the block timestamp as there is also the require statement that throws `MAX_DELAY_TIME`
the timestamp distance between blocks isn't spec'ed and is instead left for each node to decide, but we could add a test that covers if for example each block skews by `n seconds`

Reply:
add a test for delay time